external/fv3fit/fv3fit/pytorch/cyclegan/modules.py

```diff
@@ -29,6 +29,19 @@ def no_activation():
     return nn.Identity()
 
 
+class GeographicBias(nn.Module):
+    """
+    Adds a trainable bias vector of shape [6, channels, nx, ny] to the layer input.
```

Comment:
Where does this hard-coded value of 6 come from? Should it be parametrized?

Reply:
6 is the number of tiles on a cubed sphere. Since this is a module to add a bias which is resolved on the cubed sphere, I thought it should be fine for the implementation to require a tile dimension.

I've added a short statement to the docstring.